COLLABORATOR_GUIDE.md

```diff
@@ -227,3 +228,44 @@ messages. However, you are only allowed to force push to any Node.js
 branch within 10 minutes from your original push. If someone else
 pushes to the branch or the 10 minute period passes, consider the
 commit final.
+
+### Long Term Support
+
+#### What is *LTS*?
+
```

Comment:
It should not be italicized here and elsewhere. It should only be italicized in the phrase "often referred to as *LTS*". Yeah "words-as-words" is hard.